snap-server.cabal

```diff
@@ -140,7 +140,7 @@ Library
   if flag(portable) || os(windows)
     cpp-options: -DPORTABLE
   else
-    build-depends: unix                         < 2.8
+    build-depends: unix                         < 2.9
```

Comment:
Is `unix-2.8` picked up yet?  (Check output of `cabal-plan`.)  It is so that `process` so far limits `unix < 2.8`.

Reply:
It wasn't picked up, and as you mention, it doesn't build with unix-2.8. So I removed that change from this PR.